scripts/03-advanced-static-block/index.js

```diff
@@ -37,7 +37,7 @@ registerBlockType("gew/advanced-static-block", {
     }
   },
 
-  edit({ className, attributes, setAttributes }) {
+  edit({ attributes, setAttributes, isSelected, className }) {
```

Comment:
Good catch, I think there's another way to fix it. isSelected is not usefull anymore to show/hide inspector controls, it's done automatically, we can just drop the test.

Reply:
good to know, indeed if it's not useful anymore. Did not know about that 👇